packages/superset-ui-chart-controls/src/sections/advancedAnalytics.tsx

```diff
@@ -57,6 +57,9 @@ export const advancedAnalyticsControls: ControlPanelSectionConfig = {
             'Defines the size of the rolling window function, ' +
               'relative to the time granularity selected',
           ),
+          visibility: ({ controls }) =>
+            Boolean(controls?.rolling_type?.value) &&
+            controls?.rolling_type?.value !== RollingType.Cumsum,
```

Comment:
nit, shouldn't be required as we already check that `value` is truthy: 
```suggestion
            controls.rolling_type.value !== RollingType.Cumsum,
```